docs/dataset/formatting-your-datasets.md

```diff
@@ -169,7 +197,7 @@ image_to_metadata = defaultdict(dict)
 
 for record in df.itertuples():
     coords = (float(record.min_x), float(record.min_y)), (float(record.max_x), float(record.max_y))
-    bounding_box = BoundingBox(*coords)
+    bounding_box = LabeledBoundingBox(*coords, record.label)
```

Comment:
Unrelated to this PR specifically, but the s3 locator above seems to have changed.